src/flatpak-spawn.c

```diff
@@ -370,6 +370,28 @@ sandbox_flag_callback (G_GNUC_UNUSED const gchar *option_name,
   return FALSE;
 }
 
+static GPtrArray *sandbox_a11y_own_names = NULL;
+
+static gboolean
+sandbox_a11y_own_name_callback (G_GNUC_UNUSED const gchar *option_name,
+                                const gchar *value,
+                                G_GNUC_UNUSED gpointer data,
+                                GError **error)
+{
+  if (sandbox_a11y_own_names == NULL)
+    sandbox_a11y_own_names = g_ptr_array_new ();
+
+  if (!g_dbus_is_name (value) || g_dbus_is_unique_name (value))
```

Comment:
```suggestion
  if (!g_dbus_is_name (value))
```

> Checks if string is a valid D-Bus bus name (either unique or well-known).

Reply:
My intention with this check was to only allow well-known bus names (and no unique names). Sadly there is no `g_dbus_is_well_known_name()`.

Reply:
Ahh, makes more sense. 👍